src/vila/predictors.py

```diff
@@ -58,10 +58,11 @@ def normalize_bbox(
 
     # Right now only execute this for only "large" PDFs
     # TODO: Change it for all PDFs
-    if page_width > target_width or page_height > target_height:
-
+    if page_width > target_width:
         x1 = float(x1) / page_width * target_width
         x2 = float(x2) / page_width * target_width
+
+    if page_height > target_height:
```

Comment:
Does it matter that this will result in disproportionate scaling? width could reduce by a greater factor than height, or vice versa. That's guaranteed to be true if only one of these `if` clauses is True, and seems very very likely even if they are both true, unless your target_width / heights match the aspect ratio of the source document.

Reply:
That's a good point

Reply:
Maybe I should further change the scaling logic

Reply:
If you want to preserve the aspect ratio I think you can do something like

```python
if page_width > target_width or page_height > target_height:
    scale_factor = target_width / page_width if page_width > page_height else target_height / page_height

    x1 = float(x1)
    x2 = float(x2)
    y1 = float(y1)
    y2 = float(62)
```

Reply:
(Though again I don't know what the consequences are of changing the aspect ratio)